dataland-message-queue-utils/src/main/kotlin/org/dataland/datalandmessagequeueutils/cloudevents/CloudEventMessageHandler.kt

```diff
@@ -0,0 +1,68 @@
+package org.dataland.datalandmessagequeueutils.cloudevents
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.amqp.rabbit.core.RabbitTemplate
+import org.springframework.amqp.support.converter.MessagingMessageConverter
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.cloud.function.cloudevent.CloudEventMessageBuilder
+import org.springframework.cloud.function.cloudevent.CloudEventMessageUtils
+import org.springframework.messaging.MessageHeaders
+import org.springframework.stereotype.Component
+import org.springframework.util.MimeTypeUtils
+import org.springframework.amqp.core.Message as MessageMQ
+import org.springframework.amqp.core.MessageProperties as AMQPMessageProperties
+import org.springframework.messaging.Message as MessageResult
+
+/**
+ * This class ensures that the errorResponse is mapped as the default response
+ * for non-explicitly set response codes as suggested in the swagger-docs
+ * (ref https://swagger.io/docs/specification/describing-responses/)
+ */
+
+/**
+ * Handling of messages in the CloudEvents format
+ * @param rabbitTemplate
+ * @param objectMapper
+ */
+
+@Component("CloudEventMessageHandler")
+class CloudEventMessageHandler(
+    private val rabbitTemplate: RabbitTemplate,
+    @Autowired var objectMapper: ObjectMapper,
+) {
+    var converter: MessagingMessageConverter? = MessagingMessageConverter()
+    private fun constructCEMessage(input: String, type: String, correlationId: String): MessageMQ {
```

Comment:
why is it buildCEMessageAndSend, but constructCEMessage? Shouldn't we stay consistent?

Reply:
ok